### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by TomAbel**
Added some relevant info to the dull readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+Automatically generate a simple sign-up form and add new subscribers to Smaily subscribers list.
+## Features
+### Wordpress Newsletter Subscription
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+### Subscription Widget
+- Option for custom advanced HTML form.
+- Functional with NoScript users.
+- Send custom form values to Smaily.
+- Possiblility to place multiple widgets.
```
Comment (kaittodesk) [inline README.md]: I am not so sure about this feature. Was it fixed in the new version?

File: readme.txt [hunk 2]
```diff
@@ -9,16 +9,41 @@ Smaily newsletter subscription plugin for WordPress
...
+**Wordpress Newsletter Subscribers**
+
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+
+**Subscription Widget**
+- Option for custom advanced HTML form.
+- Functional even with NoScript users.
+- Supports sending custom form values to Smaily.
+- Possible to add widget anywhere.
```
Comment (kaittodesk) [inline readme.txt]: You have "Possible to add widget anywhere." here, but on in README.md. Please add it there as well or remove it from readme.txt.

File: readme.txt [hunk 3]
```diff
@@ -9,16 +11,60 @@ Smaily newsletter subscription plugin for WordPress
...
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+
+**Subscription Widget**
+
+- Option for custom advanced HTML form.
+- Functional even with NoScript users.
+- Supports sending custom form values to Smaily.
+
+= Requirements =
```
Comment (kaittodesk) [inline readme.txt]: You can remove "Requirements" section. It is already covered by `Requires PHP` and `Requires at least` properties.

File: readme.txt [hunk 4]
```diff
@@ -9,16 +11,60 @@ Smaily newsletter subscription plugin for WordPress
...
 
 == Installation ==
 
-* Install as regular plugin.
-* Open smaily plugin settings from admin side panel menu "Smaily".
-* Validate your credentials.
-* Go to Appearance -> Widgets.
-* Place "Smaily newsletter subscription" widget where you wish.
-* Using advanced tab in smaily settings will render advanced form.
+1. Upload or extract the `sendsmaily-wordpress-plugin` folder to your site's `/wp-content/plugins/` directory. You can also use the *Add new- option found in the *Plugins- menu in WordPress.
```
Comment (kaittodesk) [inline readme.txt]: Not sure what `*Add new-` and `*Plugins-` represent. Are they typos? Should they be `*Add new*` and `*Plugins*`?

File: readme.txt [hunk 5]
```diff
@@ -9,16 +11,60 @@ Smaily newsletter subscription plugin for WordPress
...
+2. Activate the plugin from the *Plugins- menu in WordPress.
+
+== Frequently Asked Questions ==
+
+= How to set up widget for signup form? =
+
+1. Validate your smaily credentials in Smaily settings menu.
```
Comment (kaittodesk) [inline readme.txt]: Be consistent. `smaily` should be written with a capital letter.

File: readme.txt [hunk 6]
```diff
@@ -9,16 +11,60 @@ Smaily newsletter subscription plugin for WordPress
...
+4. Select Title for your subscribe newsletter form.
+
+If you have added Form Submitted automation trigger from Smaily site under Automation tab you can see all availabe autoresponders in your widget settings.
+
+There is no need to select autoresponder for widget form, but if you want to customize different approach from opt-in automation trigger you can do that.
+When no autoresponder selected regular opt-in workflow will run. You can add delay, filter by field and send email after subscription. For that edit settings in Smaily automation page.
+
+== Screenshots ==
+
+1. Smaily plugin in admin view.
```
Comment (kaittodesk) [inline readme.txt]: It would be great if you could add screenshot of admin view after credentials have been validated.

File: readme.txt (hunk 4 above)
Comment (TomAbel) [reply readme.txt]: Should be the latter, for emphasis, yes. I was going for bold, added **

File: readme.txt (hunk 5 above)
Comment (kaittodesk) [reply readme.txt]: You missed this.

**Pull request merged**